python/cudf/cudf/tests/test_pickling.py

```diff
@@ -85,6 +135,26 @@ def test_sizeof_dataframe():
     assert serialized_nbytes >= sizeof
 
 
+def test_sizeof_packed_dataframe():
+    np.random.seed(0)
+    df = DataFrame()
+    nelem = 1000
+    df["keys"] = hkeys = np.arange(nelem, dtype=np.float64)
+    df["vals"] = hvals = np.random.random(nelem)
+    packed = pack(df)
+
+    nbytes = hkeys.nbytes + hvals.nbytes
+    sizeof = sys.getsizeof(packed)
+    assert sizeof < nbytes
```

Comment:
I'm just copying the checks done by `test_sizeof_dataframe` here, but is there actually a good size comparison we could do between the DataFrame columns and packed columns? Just asserting that the packed columns is smaller seems trivial.